src/lib/swapper/LifiSwapper/getUsdRate/getUsdRate.ts

```diff
@@ -0,0 +1,23 @@
+import type { ChainKey, LifiError, Token } from '@lifi/sdk'
+import type { Asset } from '@shapeshiftoss/asset-service'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { SwapError, SwapErrorType } from '@shapeshiftoss/swapper'
+import { getLifi } from 'lib/swapper/LifiSwapper/utils/getLifi'
+
+export async function getUsdRate(
+  asset: Asset,
+  lifiChainMap: Map<ChainId, ChainKey>,
+): Promise<string> {
+  const chainKey = lifiChainMap.get(asset.chainId)
+
+  if (chainKey === undefined) return '0'
```

Comment:
I wish we could return `undefined` here, but alas the `Swapper` interface does not allow it.